### PR TITLE
Initial submission of 2FA documentation.

### LINKED ISSUE #543: Add Docs for 2FA
Add documents to indicate how to enable 2FA on an Obsidian Account

### PR DESCRIPTION
**Pull request by sigrunixia**
Closes #543

### CODE REVIEW

File: en/Obsidian/2-factor authentication.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Obsidian offers users who create an [Obsidian account](https://obsidian.md/account) the option to enable 2-factor authentication (2FA).
+
+> [!info] Browser Setup Required
+> 2FA cannot be enabled from within the Obsidian application while logged in. You will need to use a web browser to add or remove 2FA.
+
+## Adding 2FA to your account
```
Comment (kepano) [inline en/Obsidian/2-factor authentication.md:6]: "Add"

File: en/Obsidian/2-factor authentication.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+> - [iCloud Keychain](https://support.apple.com/en-gb/guide/iphone/ipha6173c19f/ios)
+
+- Once connected, your authenticator app will provide you a six digit code. Enter that code below the **QR Code/Setup Key**. 
+- Finally, enter your current password.
+- Select **Complete set up**.
+- The pop-up window will be replaced with a confirmation window. 
+
+Your 2FA is now set up.
+
+## Disabling 2FA from your account
```
Comment (kepano) [inline en/Obsidian/2-factor authentication.md:26]: "Disable 2FA for your account"

File: en/Obsidian/2-factor authentication.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+Obsidian offers users who create an [Obsidian account](https://obsidian.md/account) the option to enable 2-factor authentication (2FA).
```
Comment (kepano) [inline en/Obsidian/2-factor authentication.md:1]: Add `aliases: 2FA`

File: en/Obsidian/2-factor authentication.md (hunk 1 above)
Comment (sigrunixia) [reply en/Obsidian/2-factor authentication.md:6]: Done

**Pull request merged**